new-features/index.html

```diff
@@ -110,56 +110,39 @@
-                        instantiating a widget instance and then setting the status property (ex: <em>uioutput.status = "message"</em>).</li>
-                    <li>Added support for user-installed and configured nbextensions.</li>
-                    <li>New libraries have been added to the Python 3.7 environment. These include igv, cyjupyter,
-                        genefab, humanfriendly and ndex2.</li>
+                    <li>Users can now define and import UI Builder widgets within Python modules. To make use of this feature, either upload a Python file using the <em>Upload</em> button in the upper right-hand corner of the <em>Files</em> tab or create new Python script via the <em>New</em> button in the upper right-hand corner of the <em>Files</em> tab. The Python script can then be loaded and run inside a new or existing notebook using the following syntax <pre>%run myPyScript.py</pre> The included UI Builder widget will then also be available for reuse within that notebook via the <em>Notebook</em> tab in the tool manager; which can be accessed from either the <em><i class="fa-th fa"></i> Tools</em> button in the notebook toolbar, or the <i class="fa-th fa"></i> button which is available on the left of the screen when a GenePattern cell is selected.</li>
+                    <li>Widget and parameter descriptions, for both the UI Builder and GenePattern widgets, now support HTML formatting.</li>
+                    <li>The UI Builder now supports the ability to change what tab a tool appears in, in the tool manager. (The default is either “Notebook” or the name of the module/file the tool was imported from.) To learn how to use the origin override, please see the <em><a href="/programmatic/#14otheroptions"> UI Builder> Other Options</a></em> section of the Programmatic User Guide.</li> 
                 </ul>
 
                 <h2>Administrative & Miscellaneous Features</h2>
                 <ul>
-                    <li>Added support for Jupyter 5.7.8, ipywidgets 7.4.2 and JupyterHub 1.0.</li>
+                    <li>The Notebook workspace has been updated to <a href="https://jupyter-notebook.readthedocs.io/en/stable/changelog.html", target="_blank">Jupyter 6.</a></li>
+                    <li>UI Builder widgets with no parameters will now be shown in a condensed display.</li>
+                    <li>Support for Python 3.5 has been dropped.</li>
+                    <li>The rich text widget has been updated to <a href="https://www.tiny.cloud/docs/release-notes/release-notes5013/", target="_blank">TinyMCE 5</a></li>
+                    <li>Updated to <a href="https://medium.com/plotly/plotly-py-4-0-is-here-offline-only-express-first-displayable-anywhere-fc444e5659ee", target="_blank">Plotly 4.0</a> in the Python 3.7 kernel.</li>
+                    <li><a href="/basic-features/#template-cells">Template cells</a> have been moved to <i class="fa-plus fa"></i> tab in the tool manager.</li>
```

Comment:
Similarly as with the UI builder, if the relative path ought to work then that's fine. Just leaving this comment here in case we need absolute paths in the URLs.

Reply:
Same - it only works live

Reply:
actually - wait

Reply:
I think I missed some "../"

Reply:
Should it be `<a href="../basic-features/#template-cells">`?

Reply:
ok no I think it has to be live

Reply:
No because http://genepattern-notebook.org/new-features/ points to /basic-features/ in line 118
<h2><a href="/basic-features/">General Features</a></h2>

Reply:
And that is live

Reply:
Noted! Then These issues were created due to my ignorance! Everything else rendered correctly.